out/css/spinner.css

```
@@ -0,0 +1,26 @@
+#cc-spinner {
+  display: inline-block;
+  width: 64px;
+  height: 64px;
+}
+
+#cc-spinner:after {
+  content: " ";
+  display: block;
+  width: 46px;
+  height: 46px;
+  margin: 1px;
+  border-radius: 50%;
+  border: 5px solid #009EC0;
+  border-color: #009EC0 transparent #009EC0 transparent;
```

If you use `border-style` instead, you don't have to repeat the color (so it's easier to maintain).
```suggestion
  border-style: solid none;
```